strapi/strapi
# feat(content-releases):  Create the page header with create release action

### What does it do?

Create the Release page Heading with the title, number of releases as subtitle and the button to create a new release.
Also it implements the dialog to create a new release

### Why is it needed?

To show the initial information on the Release page and add the flow to create a release

### How to test it?

The release page header should have
 - A title “Releases”
 - Subtitle displays the number of releases or 0 (now it is set to 0)
 - A button to create a new release
   - The button opens a modal with a form
   - The form requires a name
   - onSubmit the form creates a new release
